.env

```diff
@@ -22,3 +22,7 @@ SUBSCRIPTION_STELLIO_URL=http://localhost:8080
 APPLICATION_TENANTS_0_ISSUER=https://sso.eglobalmark.com/auth/realms/stellio
 APPLICATION_TENANTS_0_NAME=urn:ngsi-ld:tenant:default
 APPLICATION_TENANTS_0_DBSCHEMA=public
+
+# Pagination config for query resources endpoints
+LIMIT_DEFAULT=30
+LIMIT_MAX=100
```

Comment:
give them the same name as in docker-compose (and in the properties file in the code). no need to add some complexity learning a new name for something that already exists.